include/vcpkg/base/messages.h

```diff
@@ -2091,5 +2091,168 @@ namespace vcpkg
+    DECLARE_MESSAGE(PortBugFoundCopyrightFiles, (), "", "The following files are potential copyright files:");
+    DECLARE_MESSAGE(PortBugFoundExeInBinDir,
+                    (),
+                    "",
+                    "The following EXEs were found in /bin or /debug/bin. EXEs are not valid distribution targets.");
+    DECLARE_MESSAGE(PortBugDllsWithNoExports, (), "", "The following DLLs have no exports:");
+    DECLARE_MESSAGE(PortBugSetDllsWithoutExports,
+                    (msg::env_var),
+                    "",
+                    "DLLs without any exports are likely a bug in the build script. If this is intended, add the "
+                    "following line in the portfile:\n"
+                    "SET({env_var} enabled)");
+    DECLARE_MESSAGE(RunningCommandFailed, (msg::command_line), "", "Running command:{command_line}\nFailed:");
+    DECLARE_MESSAGE(PortBugDllAppContainerBitNotSet,
+                    (),
+                    "",
+                    "The following DLLs do not have the App Container bit set:");
+    DECLARE_MESSAGE(PortBugRequiredDllAppContainerBit,
+                    (),
+                    "",
+                    "The App Container bit must be set for Windows Store apps.");
+    DECLARE_MESSAGE(BuiltWithIncorrectArchitecture,
+                    (),
+                    "",
+                    "The following files were built for an incorrect architecture:");
+    DECLARE_MESSAGE(BinaryWithInvalidArchitecture,
+                    (msg::path, msg::expected, msg::actual),
+                    "{expected} and {actual} are architectures",
+                    "{path}\n Expected: {expected}, but was {actual}");
+    DECLARE_MESSAGE(ExpectedExtension,
+                    (msg::extension, msg::path),
+                    "",
+                    "The file extension was not {extension}: {path}");
+    DECLARE_MESSAGE(FailedToDetermineArchitecture,
+                    (msg::path, msg::command_line),
+                    "",
+                    "unable to determine the architecture of {path}.\n{command_line}");
+    DECLARE_MESSAGE(PortBugFoundDllInStaticBuild,
+                    (),
+                    "",
+                    "DLLs should not be present in a static build, but the following DLLs were found:");
+    DECLARE_MESSAGE(PortBugMismatchedNumberOfBinaries, (), "", "Mismatching number of debug and release binaries.");
+    DECLARE_MESSAGE(PortBugFoundDebugBinaries, (msg::count), "", "Found {count} debug binaries:");
+    DECLARE_MESSAGE(PortBugFoundReleaseBinaries, (msg::count), "", "Found {count} release binaries:");
+    DECLARE_MESSAGE(PortBugMissingDebugBinaries, (), "", "Debug binaries were not found.");
+    DECLARE_MESSAGE(PortBugMissingReleaseBinaries, (), "", "Release binaries were not found.");
+    DECLARE_MESSAGE(PortBugMissingImportedLibs, (msg::path), "", "Import libs were not present in {path}");
```

Comment:
```suggestion
    DECLARE_MESSAGE(PortBugMissingImportedLibs, (msg::path), "", "Import libraries were not present in {path}");
```

I know this is what the product code already said but "libs" seems likely to be mistranslated to me?